inputs/templates/test/MainVcfQc/MainVcfQc_gq_filter.json.tmpl

```diff
@@ -0,0 +1,31 @@
+{
+  "MainVcfQc.primary_contigs_fai": {{ reference_resources.primary_contigs_fai | tojson }},
+
+  "MainVcfQc.site_level_comparison_datasets": [
+    {{ reference_resources.ccdg_abel_site_level_benchmarking_dataset | tojson }},
+    {{ reference_resources.gnomad_v2_collins_site_level_benchmarking_dataset | tojson }},
+    {{ reference_resources.hgsv_byrska_bishop_site_level_benchmarking_dataset | tojson }},
+    {{ reference_resources.thousand_genomes_site_level_benchmarking_dataset | tojson }}
+  ],
+  "MainVcfQc.sample_level_comparison_datasets": [
```

Comment:
Can you make this reference the values JSON - similar to the site-level ones above?

Reply:
Done